src/sgame/sg_cmds.cpp

```diff
@@ -3853,19 +3854,19 @@ void Cmd_ListMaps_f( gentity_t *ent )
 			   }
 	);
 
-	int rows = ( mapNamesCount + 2 ) / 3;
+	int rows = ( mapNamesCount + MAX_MAPLIST_COLS - 1 ) / MAX_MAPLIST_COLS;
```

Comment:
Please add a comment here about why the MAX_MAPLIST_COLS - 1 is necessary.

Reply:
done